src/components/AddItem.js

```diff
@@ -24,9 +27,27 @@ const AddItem = ({ token }) => {
         });
   };
 
-  const handleSubmit = (event) => {
+  const handleSubmit = async (event) => {
     event.preventDefault();
```

Comment:
maybe hide the error message at the beginning of the submit. That way if the error is already shown, it will hide on submit, so we know we're getting a new error message.